translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+"\n"
+"\t\t# Update a container's image; spec.containers[*].name is required "
+"because it's a merge key\n"
+"\t\tkubectl patch pod valid-pod -p '{\"spec\":{\"containers\":[{\"name\":"
+"\"kubernetes-serve-hostname\",\"image\":\"new image\"}]}}'\n"
+"\n"
+"\t\t# Update a container's image using a json patch with positional arrays\n"
+"\t\tkubectl patch pod valid-pod --type='json' -p='[{\"op\": \"replace\", "
+"\"path\": \"/spec/containers/0/image\", \"value\":\"new image\"}]'"
+msgstr ""
+"\n"
+"\t\t# Atualiza parcialmente um node utilizando a estratégia merge patch\n"
+"\t\tkubectl patch node k8s-node-1 -p ‘{“spec”:{“unschedulable”:true}}’\n"
+"\n"
+"\t\t# Atualiza parcialmente um node identificado pelo tipo e nome no arquivo "
+"“node.json” utilizando a estratégia merge patch\n"
+"\t\tkubectl patch -f node.json -p ‘{“spec”:{“unschedulable”:true}}’\n"
+"\n"
+"\t\t# Atualiza uma image um container; spec.containers[*].name é requerido "
+"porque é a chance de merge\n"
+"\t\tkubectl patch pod valid-pod -p ‘{“spec”:{“containers”:"
+"[{“name”:”kubernetes-serve-hostname”,”image”:”new image”}]}}’\n"
+"\n"
+"\t\t# Atualiza uma image um container utilizando o json patch com positional "
+"arrays\n"
+"\t\tkubectl patch pod valid-pod —type=‘json’ -p=‘[{“op”: “replace”, “path”: "
+"“/spec/containers/0/image”, “value”:”new image”}]’"
+
+#: pkg/kubectl/cmd/options.go:29
+msgid ""
+"\n"
+"\t\t# Print flags inherited by all commands\n"
+"\t\tkubectl options"
+msgstr ""
+"\n"
+"\t\t# Mostra as opções herdadas por todos os comandos\n"
+"\t\tkubectl options"
+
+#: pkg/kubectl/cmd/clusterinfo.go:41
+msgid ""
+"\n"
+"\t\t# Print the address of the master and cluster services\n"
+"\t\tkubectl cluster-info"
+msgstr ""
+"\n"
+"\t\t# Mostra o endereço do master e dos serviços do cluster\n"
+"\t\tkubectl cluster-info"
+
+#: pkg/kubectl/cmd/version.go:32
+msgid ""
+"\n"
+"\t\t# Print the client and server versions for the current context\n"
+"\t\tkubectl version"
+msgstr ""
+"\n"
+"\t\t# Imprime a versão do cliente e do servidor para o contexto corrente\n"
+"\t\tkubectl version"
+
+#: pkg/kubectl/cmd/apiversions.go:34
+msgid ""
+"\n"
+"\t\t# Print the supported API versions\n"
+"\t\tkubectl api-versions"
+msgstr ""
+"\n"
+"\t\t# Mostra as versões de API suportadas\n"
+"\t\tkubectl api-versions"
+
+#: pkg/kubectl/cmd/replace.go:50
+msgid ""
+"\n"
+"\t\t# Replace a pod using the data in pod.json.\n"
+"\t\tkubectl replace -f ./pod.json\n"
+"\n"
+"\t\t# Replace a pod based on the JSON passed into stdin.\n"
+"\t\tcat pod.json | kubectl replace -f -\n"
+"\n"
+"\t\t# Update a single-container pod's image version (tag) to v4\n"
+"\t\tkubectl get pod mypod -o yaml | sed 's/\\(image: myimage\\):.*$/:v4/' | "
+"kubectl replace -f -\n"
+"\n"
+"\t\t# Force replace, delete and then re-create the resource\n"
+"\t\tkubectl replace --force -f ./pod.json"
+msgstr ""
+"\n"
+"\t\t# Troca um pod utlizando os dados contidos em pod.json.\n"
```

Comment:
Maybe "substitui" instead of "troca", but only a nit. Serves for the lines below either, if accepted :)